README.md

```diff
@@ -56,3 +56,77 @@ $ python setup.py develop
 ```
 ./run_tests.sh
 ```
+
+## Running a Match
+- First of all, make sure that you placed your strategy class file inside the `game/` directory.
+- You can test your own strategy by playing against any of the built-in strategies or any other third party strategy by running a match.
+- To run a new match you should make use of the `run_game.py` script, also placed inside the `game/` directory
+- For this, open the file `run_game.py` on your favorite text editor (vim, for instance :P), and add your strategy class to the list of imports, just as the example below:
+```
+from my_super_duper_strategy import MySuperDuperStrategyClass
+```
+- After that, you'll need to add your strategy to the loockup_strategies dictionary, for which we recommend to follow the same convention:
+```
+loockup_strategies = {
+		(...)
+    'my_super_duper_strategy': MySuperDuperStrategy
+}
+```
+- If everything went OK, now you should be able to play a match against any of the strategies placed inside the loockup dictionary, so let's play a match!
+- In order to run a match, you'll need to provide both strategies names as arguments, followed by the rate at which you want each move to be done, let's see an example:
+```
+alexis@alexis-ThinkPad:~/cuatro_en_linea/game$ python run_game.py --player-one closest_to_win_column --player-two my_super_duper_strategy --rate 0.5
+```
+- This will spawn a new game, and you'll be able to see the evolution of the game turn by turn. An example of what is shown in the middle of a gameplay is shown above:
+```
+None None None None None None None
+None None None None None None None
+None None None None None None None
+W    None None None None None None
+W    None None None None None None
+W    B    B    None None None None
+
+```
+- After the game has finished you'll see a message with the result of the match
+
+## Running a Tournament
+- You can run a tournament between an arbitrary number of strategies.
+- It's possible to define which strategies will be involved on the tournament by opening the `run_tournament.py` file on yor text editor and modifying the `strategies` tuple by adding/removing the strategies classes what you want to participate. Let's take a look at the example below:
+```
+strategies = (
+	ClosestToWinColumn,
+	MySuperDuperStrategy,
+	TheBestStrategyEver
+)
+```
+- A minimum of three strategies are needed in order to run a tournament, an exception will be raised otherwise
+- After filling the list with the desired strategies to play, save the file and run the script in the following way:
+```
+alexis@alexis-ThinkPad:~/cuatro_en_linea/game$ python run_tournament.py
```

Comment:
As a general comment, you usually omit the machine details. So this would be like:
```
$ python ./game/run_tournament.py
```
or if that has path troubles:
```
$ cd game/
$ python run_tournament.py
```